src/statespace/dart/SO3Joint.cpp

```diff
@@ -12,29 +12,29 @@ namespace statespace {
 namespace dart {
 
 //==============================================================================
-SO3Joint::SO3Joint(::dart::dynamics::BallJoint* _joint)
-  : SO3(), JointStateSpace(_joint)
+SO3Joint::SO3Joint(const ::dart::dynamics::BallJoint* joint)
+  : SO3(), JointStateSpace(joint)
 {
   // Do nothing.
 }
 
 //==============================================================================
 void SO3Joint::convertPositionsToState(
-    const Eigen::VectorXd& _positions, StateSpace::State* _state) const
+    const Eigen::VectorXd& positions, StateSpace::State* state) const
 {
   setQuaternion(
-      static_cast<State*>(_state),
-      SO3::Quaternion(BallJoint::convertToRotation(_positions)));
+      static_cast<State*>(state),
+      SO3::Quaternion(BallJoint::convertToRotation(positions)));
 }
 
 //==============================================================================
 void SO3Joint::convertStateToPositions(
-    const StateSpace::State* _state, Eigen::VectorXd& _positions) const
+    const StateSpace::State* state, Eigen::VectorXd& positions) const
 {
   // TODO: We should call BallJoint::convertToRotation instead of logMap once
```

Comment:
@jslee02 Do you know if this is still an issue?

Reply:
Not anymore! Resolved by https://github.com/dartsim/dart/commit/9f2909488dced6c284e35bacf1a76f9bb598d147 and c019ce5.